File: app/src/main/java/com/notonly/calendar/domain/HistoryResponse.java
package com.notonly.calendar.domain;

import java.io.Serializable;
import java.util.List;

/**
 * HistoryResponse
 * Created by wangzhen on 2020/6/13.
 */
public class HistoryResponse extends BaseBean {
    public int code;
    public String msg;
    public List<DataBean> data;

    public boolean isSuccess() {
        return code == 1;
    }

    public static class DataBean implements Serializable {
        public String picUrl;
        public String title;
        public String year;
        public String month;
        public String day;
        public String details;
    }
}

File: app/src/main/java/com/notonly/calendar/domain/HistoryResponse.kt
package com.notonly.calendar.domain

import java.io.Serializable

/**
 * HistoryResponse
 * Created by wangzhen on 2020/6/13.
 */
class HistoryResponse : BaseBean() {
    var code = 0
    var msg: String? = null
    var data: List<DataBean>? = null
    val isSuccess: Boolean
        get() = code == 1

    class DataBean : Serializable {
        var picUrl: String? = null
        var title: String? = null
        var year: String? = null
        var month: String? = null
        var day: String? = null
        var details: String? = null
    }
}